tests/perf/srv_perf.py

```diff
@@ -0,0 +1,120 @@
+# This script can be used for performance testing of the DeepRank2 package, using the SRV query classes.
+import glob
+import os
+import time
+from os import listdir
+from os.path import isfile, join
+
+import numpy
+import pandas as pd
+
+from deeprank2.domain.aminoacidlist import (alanine, arginine, asparagine,
+                                            aspartate, cysteine, glutamate,
+                                            glutamine, glycine, histidine,
+                                            isoleucine, leucine, lysine,
+                                            methionine, phenylalanine, proline,
+                                            serine, threonine, tryptophan,
+                                            tyrosine, valine)
+from deeprank2.features import (components, contact, exposure, irc,
+                                secondary_structure, surfacearea)
+from deeprank2.query import QueryCollection, SingleResidueVariantResidueQuery
+from deeprank2.utils.grid import GridSettings, MapMethod
+
+aa_dict = {"ALA": alanine, "CYS": cysteine, "ASP": aspartate,
+           "GLU": glutamate, "PHE": phenylalanine, "GLY": glycine,
+           "HIS": histidine, "ILE": isoleucine, "LYS": lysine,
+           "LEU": leucine, "MET": methionine, "ASN": asparagine,
+           "PRO": proline, "GLN": glutamine, "ARG": arginine,
+           "SER": serine, "THR": threonine, "VAL": valine,
+           "TRP": tryptophan, "TYR": tyrosine
+           }
```

Comment:
I feel like we have this or similar dictionary in other places in the code base as well. Maybe we should create this dictionary in deeprank2/domain/aminoacidlist.py instead, and load it from there.

That would also make the long import in line 11-17 (and similar in other places) obsolete.

I guess this is a separate issue though (I created #503 )